src/BotService/Infrastructure/Extensions/WebHostServiceExtensions.cs

```diff
@@ -15,7 +15,11 @@ public static class WebHostServiceExtensions
     {
         public static void RunAsCustomService(this IWebHost host)
         {
-            var webHostService = new HostService(host);
+            var webHostService = new HostService(host)
+            {
+                // Set the service name to identify logs in the Windows Event Viewer
+                ServiceName = "bot-bervice",
```

Comment:
**suggestion**: Can we move this to the constructor of the **HostService** class?